### PR TITLE
refactor: clean some old code

### PR DESCRIPTION
**Pull request by okbug**

### CODE REVIEW

File: src/expr.js [hunk 1]
```diff
@@ -281,29 +277,29 @@ function parseExprList(str, ...values) {
 
   function operator(t) {
     const op = token();
-    if (op.name == 'as') return alias(t);
-    if (op.name == 'between' || op.name == 'not between') {
+    if (op.name === 'as') return alias(t);
+    if (op.name === 'between' || op.name == 'not between') {
```
Comment (JimmyDaddy) [inline src/expr.js:281]: op.name === 'not between'. Or ['between','not between'].includes(op.name) ?

File: src/spell.js [hunk 2]
```diff
@@ -463,7 +465,7 @@ function scopeDeletedAt(spell) {
   for (const condition of whereConditions) {
     let found = false;
     walkExpr(condition, ({ type, value }) => {
-      if (type == 'id' && value == deletedAt) {
+      if (type === 'id' && value == deletedAt) {
```
Comment (JimmyDaddy) [inline src/spell.js:468]:
```
value === deletedAt
```

Comment (okbug) [reply src/spell.js:468]: if value is `null` or `undefined`, use `===` will return `false`.

**Review by JimmyDaddy — APPROVED** [review]

**Review by JimmyDaddy — COMMENTED** [review]
Did you run unit test first? It seems there are some test didn't pass, BTW this PR seems skipped github checks

File: src/spell.js [hunk 3]
```diff
@@ -81,9 +81,11 @@ function parseObjectValue(value) {
  * @returns {boolean}
  */
 function isOperatorCondition(condition) {
+  const keys = Object.keys(condition);
```
Comment (cyjake) [inline src/spell.js:84]: `Object.keys(condition)` should be called after `isPlainObject(condition)` passes

Comment (cyjake) [reply src/spell.js:84]: which is the reason that causes tests to fail

**Pull request merged**